crates/cxx-qt-lib/src/types/qset/generate.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
```

Comment:
:see_no_evil: Can we use a macro for this instead?

Reply:
Also I think generating a CXX bridge from a macro doesn't work, as the C++ code generation can't do macro expansion I think :thinking:

Reply:
Right, it'd be a lot of work in the macro to make it trigger CXX inside the macro and then with the concat_ident not being stable :shrug: this was the simplest way. As it's only a tool to help the developer update the files.

Reply:
Discussed with @Be-ing another option we explored was potentially using the `build.rs` to generate the files, but this would likely confuse IDEs, look odd to git, and could be tricky if it wrote to the OUT_DIR (how would you get the module path right?) or odd if it writes to the source.

For now we'll stick to this platform specific bash script helper, hopefully the `concat_ident!` stuff becomes stable and we can go that route eventually if we could get CXX to expand as well ...